examples/others/ime/lv_example_ime_pinyin_1.c

```diff
@@ -0,0 +1,51 @@
+#include "../../lv_examples.h"
+#if LV_USE_IME_PINYIN && LV_BUILD_EXAMPLES
+
+LV_FONT_DECLARE(lv_font_source_han_sans_normal_16);
+
+static void ta_event_cb(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t * ta = lv_event_get_target(e);
+    lv_obj_t * kb = lv_event_get_user_data(e);
+
+    if(code == LV_EVENT_FOCUSED) {
+        if(lv_indev_get_type(lv_indev_get_act()) != LV_INDEV_TYPE_KEYPAD) {
+            lv_keyboard_set_textarea(kb, ta);
+            lv_obj_clear_flag(kb, LV_OBJ_FLAG_HIDDEN);
+        }
+    }
+    else if(code == LV_EVENT_READY || code == LV_EVENT_CANCEL) {
+        lv_obj_add_flag(kb, LV_OBJ_FLAG_HIDDEN);
+        lv_obj_clear_state(ta, LV_STATE_FOCUSED);
+        lv_indev_reset(NULL, ta);   /*To forget the last clicked object to make it focusable again*/
+    }
+}
+
+void lv_example_ime_pinyin_1(void)
+{
+    lv_obj_t * pinyin_ime = lv_ime_pinyin_create(lv_scr_act());
+    lv_obj_set_style_text_font(pinyin_ime, &lv_font_source_han_sans_normal_16, 0);   // set font
+    //lv_ime_pinyin_set_dict(pinyin_ime, your_dict); // Use a custom thesaurus. If it is not set, the built-in thesaurus will be used
+
+    lv_obj_t * kb = lv_keyboard_create(lv_scr_act());
+    lv_ime_pinyin_set_keyboard(pinyin_ime, kb);
+
+    /* ta1 */
+    lv_obj_t * ta1 = lv_textarea_create(lv_scr_act());
+    lv_obj_set_style_text_font(ta1, &lv_font_source_han_sans_normal_16, 0);   // set font
+    lv_obj_align(ta1, LV_ALIGN_TOP_LEFT, 10, 10);
+
+    lv_keyboard_set_textarea(kb, ta1);
+
+    lv_obj_add_event_cb(ta1, ta_event_cb, LV_EVENT_ALL, kb);
+
+    /* ta2 */
+    lv_obj_t * ta2 = lv_textarea_create(lv_scr_act());
+    lv_obj_set_style_text_font(ta2, &lv_font_source_han_sans_normal_16, 0);   // set font
```

Comment:
Can we use the [built in simsun](https://github.com/lvgl/lvgl/blob/master/lv_conf_template.h#L379) font instead?

Reply:
> 

This is a better practice! I'll test it before that, and I may need to modify the dictionary according to simsun.

Reply:
Great! I tried my best to collect the most common CJK characters in the simsun font, but I might have missed something. Please let me know if we should add some more characters to it.

Reply:
I revised the dictionary in ime_pinyin according to simsun. I think simsun is enough for the ime_pinyin example. If simsun is updated later,  the dictionary in ime_pinyin can be modified at any time.

It is not recommended that we provide users with a general Chinese character library and dictionary, because the actual use scenarios are variable.

They can customize according to their actual needs. In a similar way, I provided the method of using Chinese character library and common Chinese characters on the lvgl Chinese website I created.